scripts/generate_screenshots.py

```
@@ -0,0 +1,108 @@
+
+from rich.console import Console
+from rich.text import Text
+
+# Shared mock data (resembles real comfy cs output).
+RESULTS = [
+    {
+        "repository": "Comfy-Org/ComfyUI",
+        "file": "nodes.py",
+        "file_url": "https://github.com/Comfy-Org/ComfyUI/blob/abc123/nodes.py",
+        "matches": [
+            {"line": 1698, "preview": "class LoadImage:", "url": "https://github.com/Comfy-Org/ComfyUI/blob/abc123/nodes.py#L1698"},
```

Pylint (4.0.5)

[convention] Line too long (132/120)

(C0301)